osbs/utils.py

```diff
@@ -327,20 +328,34 @@ def make_name_from_git(repo, branch, limit=53, separator='-', hash_size=5):
 
     size = (2 * len(separator)) + hash_size
 
+    disallowed = '._'
```

Comment:
and if they would be having wrong chars in names it would fail? we are suppose to get rid of wrong chars to submit valid name